subsystem_controllers/config/guidance_controller_config.yaml

```diff
@@ -10,6 +11,13 @@
 
     # String: The namespace for nodes in this subsystem. All nodes under this namespace will have their lifecycle's managed by this controller
     subsystem_namespace: /guidance
+        
+    # List of guidance plugins that are ported to ROS2. If not in this list, it is assumed to be ROS1, and not managed.
+    ros2_initial_plugins:
```

Comment:
Isn't this parameter already in this file on develop

Reply:
this was due to the bad merge from before. fixed now